server/src/main/java/org/opensearch/identity/authz/PERMISSIONS_MODEL.md

```diff
@@ -0,0 +1,52 @@
+The pre-Identity permissions model made common use of a self-describing nomenclature which saw many permissions correlate to REST API operations.
+Despite this correlation, many REST API operations make use of numerous permissions at a time making the control of API access insufficient.
+
+With the transition to an Identity-based OpenSearch, it is productive to re-conceptualize Permissions as tokens which grant subjects increased privileges to certain data and operations.
+Keeping with the general practice of granting "least privileges" Permissions can be renovated to associate with a given subject Identity instead of being based on a cluster operation.
+Instead of Permissions primarily serving to restrict the certain operations on a cluster, the new Permission model seeks to grant individual Subject the privileges to execute operations.
+
+To further clarify this change consider the difference between the two lenses of the scenario outlined on the [OpenSearch Permission Documentation](https://opensearch.org/docs/latest/security-plugin/access-control/permissions/):
+
+Imagine that a subject `User_1` seeks to perform a `_bulk` request using the following:
+
+```
+POST _bulk
+{ "delete": { "_index": "test-index", "_id": "tt2229499" } }
+{ "index": { "_index": "test-index", "_id": "tt1979320" } }
+{ "title": "Rush", "year": 2013 }
+{ "create": { "_index": "test-index", "_id": "tt1392214" } }
+{ "title": "Prisoners", "year": 2013 }
+{ "update": { "_index": "test-index", "_id": "tt0816711" } }
+{ "doc" : { "title": "World War Z" } }
+```
+
+Under the previous Permissions model, `User_1` needed to have four separate permissions for `test_index` which all required verification each time the operation would be performed.
+
+Under the new model of Permissions, it is possible to simply check the Identity of `User_1` for the required permissions and then after authorizing the request once, the operation can be assumed authorized until the authorization token expires. This Permission model is similar to the
+authentication model introduced with the Identity changes and makes use of a Permission model that maintains a least privileges model by assuming all Subjects are permission-less upon initial authentication.
+After a Subject is authenticated, Permission tokens are only added to their Identity upon the Subjects first attempt to operate using them. Returning to the example, when `User_1` logs into OpenSearch, they are assumed to have no permissions.
+Then, when they attempt to perform the `_bulk` request, the appropriate Permissions checks are executed for each of the operations constituting their request. As each operation is approved, a decaying token is added to their Identity which signifies
+that for the life of the token, any further attempts to perform the same type of operation are automatically authorized. After a decay period, the token is reset and the subject will once again require authorization in order to perform the requests.
```

Comment:
nit: this line break is not needed and the sentence can be continued in previous line.

Also should it be `same operation` instead of `same type of operation`?

nit: `After expiry the token will be reset and the subject will once again require authorization in order to perform the operations.`

Reply:
Yeah I think same operation is more clear. I will fix this.